src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplAPIDocumentationSteps.java

```diff
@@ -0,0 +1,75 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+
+import static org.testng.Assert.assertTrue;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import gov.healthit.chpl.aqa.pageObjects.ChplAPIPage;
+
+/**
+ * Class ChplAPIDocumentationSteps definition.
+ */
+public class ChplAPIDocumentationSteps {
```

Comment:
Take a look at "ChplDownloadSteps" and "ManageDevelopersAndProductsSteps" for examples of steps pages that extend the BaseSteps class. By extending BaseSteps you don't have to manage the webdriver, url, timeouts, etc.